### DISCUSSION TITLE
Prevent auto start download from extension

### DISCUSSION THREAD

**Discussion by KuroSetsuna29** [Tzahi12345/YoutubeDL-Material#441] [General]
Is there a way to prevent the download from auto-starting download when coming from the extension? Especially when I have quality selection enabled I want to be able to pick the quality sometimes before it automatically starts downloading the video.

**Comment by GlassedSilver** [comment]
Any specific reason why you'd not rather set the quality and anything else maybe within the extension then?

Seems like the logical alternative to me. Similar to J-Downloader's extension one could have both: all options + "start paused".

Once #420 is implemented one could then go to the Download Manager and fire away all paused tasks from there then I guess because they will be properly queued internally.

Just a thought. @Tzahi12345?

**Reply by KuroSetsuna29** [reply]
That would be great as well, just having a way to download a specific quality before it starts downloading the default max quality, but as you mentioned if the quality dropdown was in the extension that would be awesome.
